apps/sms/style/sms.css

```diff
@@ -933,16 +932,29 @@ body[role="application"] form[role="dialog"][data-type="action"] > header:first-
 }
 
 /* Phone details label styles */
-.phone-type,
+.carrier-information {
+  display: flex;
+  flex-flow: row nowrap; 
+}
+
+.phone-type, .phone-carrier {
+  min-width: 0px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
I think you don't need the white-space property because it's inherited from carrier-wrapper